packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -826,6 +828,9 @@ class AreaPaneHeader extends StatelessWidget implements PreferredSizeWidget {
   }
 
   List<Widget> _buildActions() {
+    if (onlyShowRightActions) {
```

Comment:
This configuration for _buildActions was added to make sure that there is maximum space left for the `title` in small screens. (otherwise there was just a big empty gap)